yugabyte/run-jepsen.py

```diff
@@ -55,68 +65,172 @@
     "partition-half",
     "partition-one",
     "partition-ring",
-    "partition",
-    # "clock-skew",
+    "partition"
 ]
 
 SCRIPT_DIR = os.path.abspath(os.path.dirname(sys.argv[0]))
 STORE_DIR = os.path.join(SCRIPT_DIR, "store")
+LOGS_DIR = os.path.join(SCRIPT_DIR, "logs")
 SORT_RESULTS_SH = os.path.join(SCRIPT_DIR, "sort-results.sh")
 
 child_processes = []
 
+
 def cleanup():
     deadline = time.time() + 5
     for p in child_processes:
-        while p.poll() == None and time.time() < deadline:
+        while p.poll() is None and time.time() < deadline:
             time.sleep(1)
         try:
             p.kill()
         except OSError, e:
             if e.errno != errno.ESRCH:
                 raise e
 
-def run_cmd(cmd, shell=True, timeout=None, exit_on_error=True):
-    print cmd
-    p = subprocess.Popen(cmd, shell=True)
-    child_processes.append(p)
 
-    if timeout:
-        deadline = time.time() + timeout
-    while p.poll() == None and (timeout == None or time.time() < deadline):
-        time.sleep(1)
+def run_cmd(cmd, shell=True, timeout=None, exit_on_error=True, log_name_prefix=None):
+    logging.info("Running command: %s", cmd)
+    if log_name_prefix is not None:
+        timestamp_str = time.strftime('%Y-%m-%dT%H-%M-%S')
+        log_name_prefix += '_' + timestamp_str
+        stdout_path = os.path.join(LOGS_DIR, log_name_prefix + '_stdout.log')
+        stderr_path = os.path.join(LOGS_DIR, log_name_prefix + '_stderr.log')
+        logging.info("stdout log: %s, stderr log: %s", stdout_path, stderr_path)
```

Comment:
Why do we need to keep all console logs for Jepsen given that we have it in `jepsen.log` and it could be very huge? But probably it is better to tune console logger of Jepsen itself in order to reduce console logs amount...

Reply:
Made the script remove the stdout log for now.